### PR TITLE
db/seeding file

### PR DESCRIPTION
**Pull request by JensenYuen**
1. Added json files to db 
2. added code to seed file to create trails instances (50)

### CODE REVIEW

File: db/data/checklist.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    },
+    "kitchen_tools": {
+      "required": ["Backpacking stove", "Fuel", "Cookset (at least 1 pot)", "Mug/Cup", "Dish/Bowl", "Utensils", "Biodegradable soap", "Small quick-dry towel", "Collapsible water container"]
+    },
+    "food_water": {
+      "required": ["Water bottle or hydration bladder","Water filter/purifier or chemical treatment", "Energy food and drinks, trail mix","Extra day's supply of food"]
+    },
+    "clothes_footwear": {
+      "required": ["Moisture-wicking undergarments", "Moisture-wicking t-shirt (Merino wool or polyester)", "Quick-drying pants/shorts", "Long-sleeve shirt (for sun, bugs)","Lightweight fleece or jacket", "Boots or shoes", "Socks (synthetic or wool)"],
+      "cold_weather": ["Rainwear (jacket and pants)", "Long underwear", "Insulated jacket/vest (puffy)", "Fleece pants", "Gloves or mittens", "Warm hat"],
```
Comment (eztree) [inline db/data/checklist.json:15]: "snow_weather": ["Crampons", "Ice axe"],

**Review by eztree — CHANGES_REQUESTED** [review]
change it!

File: db/data/checklist.json (hunk 1 above)
Comment (JensenYuen) [reply db/data/checklist.json:15]: Added!

**Review by eztree — APPROVED** [review]
looks good!

**Pull request merged**